src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -49,6 +51,16 @@
 )
 
 
+class ReadOnlyWidget(admin.widgets.ManyToManyRawIdWidget):
+    template_name = "admin/widgets/raw_id_read_only.html"
+
+    def format_value(self, value):
+        if not value:
+            return ""
+        models = self.rel.model.objects.filter(pk__in=value)
+        return ", ".join(str(model) for model in models) if value else ""
```

Comment:
These are instances right?
```suggestion
        objects = self.rel.model.objects.filter(pk__in=value)
        return ", ".join(str(obj) for obj in objects) if value else ""
```